pkg/boot/align/align.go

```diff
@@ -1,15 +1,23 @@
 // Copyright 2015-2022 the u-root Authors. All rights reserved
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
-
 package align
 
 import "os"
 
-var pageMask = uint(os.Getpagesize() - 1)
+var pageSize = uint(os.Getpagesize())
+
+func AlignUpBySize(v uint, alignSize uint) uint {
```

Comment:
align.go:1:1: package comment should be of the form "Package align ..."
align.go:10:1: exported function AlignUpBySize should have comment or be unexported
align.go:10:6: func name will be used as align.AlignUpBySize by other packages, and that stutters; consider calling this UpBySize
align.go:19:1: exported function AlignUpPageSize should have comment or be unexported
align.go:19:6: func name will be used as align.AlignUpPageSize by other packages, and that stutters; consider calling this UpPageSize
align.go:23:1: exported function AlignUpPageSizePtr should have comment or be unexported
align.go:23:6: func name will be used as align.AlignUpPageSizePtr by other packages, and that stutters; consider calling this UpPageSizePtr

Reply:
Done